mediathread/main/views.py

```diff
@@ -787,17 +788,40 @@ def get_success_url(self):
         return reverse('course-invite-complete')
 
 
-class CourseInstructorDashboardView(LoggedInMixin, DetailView):
+class InstructorDashboardView(LoggedInMixin, DetailView):
```

Comment:
The view should check that the logged in user is faculty for the requested course.